src/components/PlayerProfile.vue

```diff
@@ -162,17 +165,19 @@ export default {
     $route: 'executeQuery',
   },
   methods: {
+    fromNow(date) {
+      return `${countdown(new Date(), date, countdown.ALL, 1)} ago`;
+    },
     formatDate(date) {
-      if (date == null) {
+      if (!date) {
         return '-';
       }
       const d = new Date(date);
-      return (d.getFullYear() === 1969) ? 'Never' : d.toLocaleString();
+      return d.toLocaleString();
     },
     getRanks() {
-      if (this.player === undefined || this.player.groups.length === 0) {
-        return 'None';
-      }
+      if (this.player === undefined || this.player.groups === undefined) return null;
```

Comment:
We're no longer showing the group section if the player isn't in a group? I think we should keep the "Groups: None" label.
Also, you can simplify this statement a bit:
`if( !this.player?.groups)`